dependencies.yaml

```
@@ -507,7 +507,7 @@ dependencies:
     common:
       - output_types: [conda]
         packages:
-          - &pytorch_conda pytorch>=2.3
+          - &pytorch_conda pytorch>=2.3,<2.5
```

Typically we spell upper pinnings with an alpha like:
```suggestion
          - &pytorch_conda pytorch>=2.3,<2.5a0
```